startup/configuration/internals/logiVehicle.sqf

```diff
@@ -0,0 +1,15 @@
+#include "macros.hpp"
+
+if (isNil "f_var_logiVehicle_already_run") then
+{
+    #include "..\..\..\logi_macros.hpp"
+    f_var_logiVehicle_already_run = true;
+
+    #include "..\..\..\configuration\logiVehicle.sqf"
+    f_var_logiVehicle_loaded = true;
```

Comment:
Is this used for anything (other than for potential debugging)?

(unrelated, but I'm also trying to find out what the "Start a review" button does)

Reply:
Turns out it's no different to adding a single comment. Huh.

Reply:
Not used for anything now but it used to be that each component that defines config vars would set a "loaded" var to true once it had finished, which wasn't always done in preinit or init.  I like to keep them around for consistency.